webapp/src/components/table/table.tsx

```diff
@@ -59,6 +95,59 @@ const Table = (props: Props) => {
         },
     }), [activeView])
 
+    const onAutoSizeColumn = ((columnID: string) => {
+        let longestSize = 0
+
+        const visibleProperties = board.cardProperties.filter(() => activeView.visiblePropertyIds.includes(columnID))
+
+        if (columnID === Constants.titleColumnId) {
+            cards.forEach((card) => {
+                const thisLen = getTextWidth(card.title, title.fontDescriptor)
+                if (thisLen > longestSize) {
+                    longestSize = thisLen
+                }
+            })
+            longestSize += title.padding
+        } else {
+            const template = visibleProperties.find((t) => t.id === columnID)
+            if (!template) {
+                return
+            }
+
+            // Set to Header size initally
+            longestSize = getTextWidth(template.name.toUpperCase(), header.fontDescriptor) + header.padding
+
+            let padding = text.padding
+            let fontDescriptor = text.fontDescriptor
+            if (template.type === 'select') {
+                padding = select.padding
+                fontDescriptor = select.fontDescriptor
+            }
+
+            cards.forEach((card) => {
+                const propertyValue = card.properties[columnID]
+                let displayValue = OctoUtils.propertyDisplayValue(card, propertyValue, template!) || ''
+                if (template.type === 'select') {
+                    displayValue = displayValue.toUpperCase()
+                }
+
+                const thisLen = getTextWidth(displayValue, fontDescriptor) + padding
+                if (thisLen > longestSize) {
+                    longestSize = thisLen
+                }
+            })
+        }
+
+        if (longestSize === 0) {
```

Comment:
I wonder if there should be a min-width for columns? Not sure if there are issues with very narrow columns currently.

Reply:
There is a minimum width. And the existing code already handles that. A completely empty column gets set to the minimum width. (100px)